easybuild/easyconfigs/b/Blender/Blender-3.3.1.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Blender'
+version = '3.3.1'
+versionsuffix = ''
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'system', 'version': ''}
+
+sources = ['blender-%(version)s-linux-x64.tar.xz']
+source_urls = ['https://ftp.nluug.nl/pub/graphics/blender/release/Blender%(version_major_minor)s/']
+patches = []
+checksums = [
+    '3089a485dd621785d7a702089aba72d07b8f733a362e901ec1449b9a379546f2'
+]
```

```suggestion
source_urls = ['http://ftp.nluug.nl/pub/graphics/%(namelower)s/release/%(name)s%(version_major_minor)s/']
sources = ['%(namelower)s-%(version)s-linux-x64.tar.xz']
checksums = ['3089a485dd621785d7a702089aba72d07b8f733a362e901ec1449b9a379546f2']
```

See also [template names/values](https://docs.easybuild.io/en/latest/version-specific/easyconfig_templates.html).